server/src/resolvers/Subscription.js

```diff
@@ -1,15 +1,15 @@
 const Subscription = {
   feedSubscription: {
-    subscribe: (parent, args, ctx, info) => {
-      // TODO: handle authenticated subscriptions
-      return ctx.db.subscription.comment(
-        {
-          where: {
-            node: {
-              isPublic: true
-            }
+    subscribe: (parent, { includePrivate }, ctx, info) => {
+      const queryPublic = !includePrivate ? {
```

Comment:
defaults to not include private comments when called without argument. 

note: snackbar subscription has not been updated to include private comments based on user auth status.